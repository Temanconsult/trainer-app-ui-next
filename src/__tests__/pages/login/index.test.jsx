import React from 'react'
import { render } from '@testing-library/react'
import Login from '../../../pages/login'

describe('Pages:Login', () => {
  it('renders the Login page', () => {
    const { asFragment } = render(<Login />)
    expect(asFragment()).toMatchSnapshot()
  })
})
