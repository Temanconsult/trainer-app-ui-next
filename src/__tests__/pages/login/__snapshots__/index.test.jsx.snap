// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages:Login renders the Login page 1`] = `
<DocumentFragment>
  <p>
    Boilerplate content for Login
  </p>
  <p>
    Try navigating to
    <a
      href="/"
    >
      the homepage
    </a>
    and observe the URL changing
  </p>
</DocumentFragment>
`;
